base/libgit2/blob.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 function content(blob::GitBlob)
-    return ccall((:git_blob_rawcontent, :libgit2), Ptr{Void}, (Ptr{Void},), blob.ptr)
+    unsafe_string(ccall((:git_blob_rawcontent, :libgit2), Ptr{UInt8}, (Ptr{Void},), blob.ptr))
```

Comment:
Yes. We can deal with the NUL issue by calling it with a length argument (which I presume is [`git_blob_rawsize`](https://libgit2.github.com/libgit2/#HEAD/group/blob/git_blob_rawsize)). But I don't know how we could deal with non-UTF8 data.